src/Education.UnitTests/XamlGenerator/RawTextProcessorTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using FluentAssertions;
```

Comment:
Forgot copyright, will add it with any other necessary changes (or standalone if there are none.)